File: easy/if-else/Python/if_else.py
"""
Given an integer,n, perform the following conditional actions:
If n is odd, print Weird
If n is even and in the inclusive range of  to , print Not Weird
If n is even and in the inclusive range of  to , print Weird
If n is even and greater than , print Not Weird
"""


def check_n(num):
    if num % 2:
        return ('Weird')
    else:
        if num >= 2 and num <= 5:
            return ('Not Weird')
        elif num >= 6 and num <= 20:
            return ('Weird')
        else:
            return ('Not Weird')


if __name__ == '__main__':
    n = int(input('What number do you want to check?').strip())
    print(check_n(n))

File: easy/if-else/JavaScript/code/if-else.ts
export function weird_checker(n: number) {
	if (n % 2) {
		return 'Weird';
	} else {
		if (n >= 2 && n <= 5) {
			return 'Weird';
		}
		if (n >= 6 && n <= 20) {
			return 'Weird';
		} else {
			return 'Not Weird';
		}
	}
}
console.log(weird_checker(3)); //Expected Output: 'Weird'
console.log(weird_checker(24)); //Expected Output: 'Not Weird'
